packages/datagateway-dataview/server/e2e-settings.json

```diff
@@ -6,9 +6,9 @@
     "datasetGetSize": false,
     "datasetGetCount": true
   },
-  "idsUrl": "https://localhost:8181/ids",
-  "apiUrl": "http://localhost:5000",
-  "downloadApiUrl": "https://localhost:8181/topcat",
+  "idsUrl": "https://scigateway-preprod.esc.rl.ac.uk:8181/ids",
+  "apiUrl": "https://scigateway-preprod.esc.rl.ac.uk/datagateway-api",
+  "downloadApiUrl": "https://scigateway-preprod.esc.rl.ac.uk:8181/topcat",
```

Comment:
Hi Goel, these changes should be reverted as Matt has now merged in work that makes use of a local ICAT stack for e2e tests. The dependency on preprod has therefore been removed and should only be used when testing locally.